tofu/main/harvester/variables.tf

```diff
@@ -0,0 +1,159 @@
+variable "ssh_public_key_path" {
+  description = "Path to SSH public key file (can be generated with `ssh-keygen -t ed25519`)"
+  default     = "~/.ssh/id_ed25519.pub"
+}
+
+variable "ssh_private_key_path" {
+  description = "Path to SSH private key file (can be generated with `ssh-keygen -t ed25519`)"
+  default     = "~/.ssh/id_ed25519"
+}
+
+variable "ssh_user" {
+  description = "User name to use for the SSH connection to all nodes in all clusters"
+  default     = "opensuse"
+}
+
+variable "ssh_bastion_host" {
+  description = "Public name of the SSH bastion host. Leave null for publicly accessible Harvester instances"
+  type        = string
+  default     = null
+}
+
+variable "ssh_bastion_user" {
+  description = "User name to connect to the SSH bastion host"
+  default     = null
+}
+
+variable "ssh_bastion_key_path" {
+  description = "Path of private ssh key used to access the bastion host to access Harvester"
+  type        = string
+  default     = null
+}
+
+variable "upstream_cluster" {
+  description = "Upstream cluster configuration. See tofu/modules/generic/test_environment/variables.tf for details"
+  type        = any
+}
+
+variable "upstream_cluster_distro_module" {
+  description = "Name of the module to use for the upstream cluster"
+  default     = "generic/k3s"
+}
+
+variable "downstream_cluster_templates" {
+  description = "List of downstream cluster configurations. See tofu/modules/generic/test_environment/variables.tf for details"
+  type        = list(any)
+}
+
+variable "downstream_cluster_distro_module" {
+  description = "Name of the module to use for the downstream clusters"
+  default     = "generic/k3s"
+}
+
+variable "tester_cluster" {
+  description = "Tester cluster configuration. See tofu/modules/generic/test_environment/variables.tf for details"
+  type        = any
+  default     = null
+}
```

Comment:
This should be a nullable field, right? Probably should add that if so.

Reply:
The default value for `nullable` on any given var is `true`, so we should only need to explicitly set it if we **don't** want a var to be nullable.